server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/src/main/scala/org/apache/james/jmap/rfc8621/contract/EchoMethodContract.scala

```diff
@@ -130,7 +130,7 @@ trait EchoMethodContract {
   }
 
   @Test
-  def echoMethodShouldReturnUnknownMethodWhenMissingCoreCapability(): Unit = {
+  def echoMethodShouldNotRequireCapability(): Unit = {
```

Comment:
I don't get this commit... is this specified somewhere in the spec?

Reply:
> I don't get this commit... is this specified somewhere in the spec?

Partly. mail capability is the only one mentioned in the mail objects.

Identity / EmailSubmission are unlocked by submission capability alone.

Here I would like to apply a standard inter-operability idiom said MUCH in the team since I joined: be relaxed about what you accept, strict on what you return. I'm just applying this mantra.